.github/workflows/development.yaml

```
@@ -0,0 +1,124 @@
+name: Development
+on:
+  pull_request:
+  push:
+    tags:
+      - '*.*.*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - py_ver: 3.8
+            distro: alpine
+            image: djbase
+    env:
+      PY_VER: ${{matrix.py_ver}}
+      DISTRO: ${{matrix.distro}}
+      IMAGE: ${{matrix.image}}
+      DOCKER_CLIENT_TIMEOUT: "120"
+      COMPOSE_HTTP_TIMEOUT: "120"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Compile image
+        run: |
+          export PKG_NAME=$(cat setup.py | awk -F\' '/pkg_name = / {print $2}')
```

Could we simply just assume the name of the repo matches the name of the PyPi package? That way it is also meaningful?

```suggestion
          export PKG_NAME=$(echo ${PWD##*/} | sed 's|-|_|g')
```

You could then update your `setup.py` to also draw from the current directory name so that it is a better template for other elements.